webpack.config.js

```diff
@@ -1,43 +1,50 @@
+/* eslint-disable import/no-nodejs-modules */
 const webpack = require( 'webpack' );
 const path = require( 'path' );
-const ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
 const autoprefixer = require( 'autoprefixer' );
+const url = require( 'postcss-url' );
+const TerserPlugin = require( 'terser-webpack-plugin' );
+const os = require( 'os' );
 
 const isProd = 'production' === process.env.NODE_ENV;
 const isI18n = 'i18n' === process.env.NODE_ENV;
+const isDev = ! isProd && ! isI18n;
 
 const browsers = 'last 2 versions, not ie_mob 10, not ie 10';
 
-process.noDeprecation = true; // see https://github.com/webpack/loader-utils/issues/56
-
-const babelSettings = {
-	presets: [
-		[ 'env', {
-			useBuiltIns: true,
-			targets: { browsers },
-			loose: true,
-			// modules: false, // add-module-exports breaks with WebPack 2 and modules: false
-		} ],
-		'stage-1',
-		'react'
-	],
-	plugins: [
-		'add-module-exports',
-		'lodash',
-		[
-			'transform-imports',
-			{
-				'state/selectors': {
-					transform: 'state/selectors/${member}',
-					kebabCase: true,
-				},
-			},
-		],
-	],
-	babelrc: false,
-};
+const cssLoaders = [
+	isDev ? 'style-loader?hmr=false' : MiniCssExtractPlugin.loader,
+	'css-loader',
+	{
+		loader: 'postcss-loader',
+		options: {
+			plugins: () => [
+				autoprefixer( { browsers } ),
+				url( {
+					url: ( asset ) => asset.url.startsWith( 'data:' ) ? asset.url : ( 'https://wordpress.com/' + asset.url ),
```

Comment:
css-loader doesn't have this functionality anymore, they recommend to use this postcss plugin instead.